optimizely-experiments/src/dist/scripts.js

```diff
@@ -0,0 +1,291 @@
+      }
+    }
+
+    if (dataSet.status === 'running') {
+      if (dataSet.description === undefined || dataSet.description.length <= 0) {
+        $('.experiment-list').append("<tr><td data-toggle=\"tooltip\" title=\"Type :  ".concat(dataSet.type, "\" class=\"details\" id=\"").concat(dataSet.id, "\">").concat(dataSet.key, "</td>\n   <td class=\"edit\"><p data-placement=\"top\" data-toggle=\"tooltip\" title=\"Pause\"><button id=\"pause\" class=\"action\" data-title=\"Pause\"><span><i class=\"fa fa-pause\"></i></span></button></p></td>\n    <td class=\"result\"><p data-placement=\"top\" data-toggle=\"tooltip\" title=\"Result\"><a target=\"_blank\" href=\"\" id=\"").concat(dataSet.campaign_id, "\"><button class=\"result-icon\"><span><i class=\"fa fa-bar-chart\"></i></span></button></p></td>\n   </tr>"));
+      } else {
+        $('.experiment-list').append("<tr><td data-toggle=\"tooltip\" title=\"Type :  ".concat(dataSet.type, " \n Description :  ").concat(dataSet.description, "\" class=\"details\" id=\"").concat(dataSet.id, "\">").concat(dataSet.key, "</td>\n    <td class=\"edit\"><p data-placement=\"top\" data-toggle=\"tooltip\" title=\"Pause\"><button id=\"pause\" class=\"action\" data-title=\"Pause\"><span><i class=\"fa fa-pause\"></i></span></button></p></td>\n   <td class=\"result\"><p data-placement=\"top\" data-toggle=\"tooltip\" title=\"Result\"><a target=\"_blank\" href=\"\" id=\"").concat(dataSet.campaign_id, "\"><button class=\"result-icon\"><span><i class=\"fa fa-bar-chart\"></i></span></button></a></p></td>\n    </tr>"));
+      }
+    } //  }
+
+  });
+  $('#loader').hide();
+  $('.container').show();
+  domChangeListnerOfExperimentsListPage();
+}
+
+function domChangeListnerOfExperimentDetailPage() {
+  $('.goBack').click(function () {
+    $('.experiment-data, .experiment-key, .audiences').empty();
+    $('.labell').css({
+      display: 'block'
+    });
+    $('.new-experiment').css({
+      display: 'block'
+    });
+    $('.table-responsive, .refresh-btn').show();
+    $('.experiment-container').hide();
+    $('.new-experiment-container').hide();
+  });
+  $('.audienceLink').click(function (e) {
+    var url = "https://app.optimizely.com/v2/projects/".concat(extensionField.config.project_id, "/audiences");
+    $(e.currentTarget).attr('href', url);
+  });
+}
+
+function domChangeListnerOfExperimentsListPage() {
+  $('.details').click(function (e) {
+    var id = e.currentTarget.id;
+    optimizely.getExperimentDetail(id).then(function (response) {
+      renderToShowExperimentDetails(response);
+    });
+    $('.delete-container, .list, .edit-container, .new-experiment-container, .refresh-btn').hide();
+    $('.labell').css({
+      display: 'none'
+    });
+    $('.new-experiment').css({
+      display: 'none'
+    });
+    $('#loader').show();
+  });
+  $('.result-icon').click(function (e) {
+    console.log(e.currentTarget, e.current);
```

Comment:
remove log

Reply:
Rest changes are reviewed. Please address the above comment
@kego1992 @Amitkanswal